app/services/commits_runner.rb

```diff
@@ -0,0 +1,25 @@
+class CommitsRunner
+  def self.run(commits)
+    commits.each do |commit|
+      if (valid?(commit))
+        create(commit)
+        BenchmarkPool.enqueue(commit[:repo][:name], commit[:sha])
+      end
+    end
+  end
+
+  private
+
+  def self.valid?(commit)
+    !Commit.merge_or_skip_ci?(commit[:message]) && Commit.valid_author?(commit[:author_name])
```

Comment:
Unless I'm missing something, this will always return `nil`, it should rather be `commit[:author][:name]`.